src/Input/CardDisplayInput.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { useMemo } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import propTypes from '@styled-system/prop-types'
+import css from '@styled-system/css'
+import { variant } from 'styled-system'
+import { Icon } from '../Icon'
+import { Typography, typography } from '../Typography'
+import { Flex, Touchable } from '@ivoryio/kogaio'
+
+import { default as variants } from './CardDisplayInput.variants'
+import { SubLabel } from './SubLabel'
+import sizes from '../assets/sizes'
+
+export const CardDisplayInput = ({
+  disabled,
+  id,

```

Comment:
please make sure the id property is not required, it will yield an unneccessary warning in console if not provided.